### PR TITLE
Added theme:Web-Nary

### PR DESCRIPTION
**Pull request by BRAVO68WEB**
## Check List

**Please read and check followings before submitting a PR.**

- [x] I want to publish my theme on Hexo official website.

    - [x] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.
    - [x] `preview` URL is correct.
    - [x] `preview` URL web site is rendered correctly.
    - [x] Add a screenshot to `source/themes/screenshots`.
    - [x] Screenshot filename is same as value of `name`.
    - [x] Screenshot size is `800 * 500`.
    - [x] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
    - Languages:
    - [x] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [ ] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by BRAVO68WEB** [comment]
Anyone Reply!!!!!

**Comment by BRAVO68WEB** [comment]
@SukkaW  Why my PR is not accepted??

**Comment by BRAVO68WEB** [comment]
@hexojs Will my theme be added not not??

File: source/_data/themes.yml [hunk 1]
```diff
@@ -3320,6 +3320,20 @@
     - widgets
     - responsive
     - dark/light
+- name: Web-Nary
+  description: A Fully Customised Hexo Flat Modular Theme for Bloging and More ...
+  link: https://github.com/BRAVO68WEB/hexo-webnary-theme/
```
Comment (stevenjoezhang) [inline source/_data/themes.yml:3325]: Would you mind renaming the theme to `hexo-theme-webnary`? Thanks.

Comment (BRAVO68WEB) [reply source/_data/themes.yml:3325]: Ok Sure

**Comment by BRAVO68WEB** [comment]
I did the change mentioned by you

**Pull request merged**